general/server/plugins/index.md

```diff
@@ -271,6 +271,46 @@ Takes your tuners in TVHeadEnd and emulates a HDHomeRun, in order to connect to
 
 * [GitHub](https://github.com/TheJF/antennas)
 
+#### Merge Versions
+
+Automatically group every repeated movie.
+
+**Links:**
+
+* [GitHub](https://github.com/danieladov/jellyfin-plugin-mergeversions)
+
+#### Intros
+
+Download flashy intros from prerolls.video for your movies.
+
+**Links:**
+
+* [GitHub](https://github.com/dkanada/jellyfin-plugin-intros)
+
+#### Youtube Metadata
```

Comment:
```suggestion
#### YouTube Metadata
```

Reply:
I think you missed this one :smile:

Reply:
Oops. Im still figuring out how pull requests work.

Reply:
No worries, thanks for the fix!